### DISCUSSION TITLE
Simple Dark Theme

### DISCUSSION THREAD

**Discussion by Jojo10011** [mathoudebine/turing-smart-screen-python#195] [Themes]
Here is a Theme I made using the Cyberpunk Colors/Font
![preview](https://user-images.githubusercontent.com/58893218/229151709-24a575c2-1ec6-4433-a9bc-9a694c7a7636.png)
 I tried to keep it simple with minimal Information displayed

**Download**
[Landscape-Material-Simple.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11122931/Landscape-Material-Simple.zip)

**Comment by ooemir23** [comment]
Bro have a add Fps metre?

**Comment by Freekers** [comment]
@Jojo10011 Nice theme! I tried creating a portrait version of it but didn't succeed so far. Would it be possible for you to create one? Thanks!

**Reply by Jojo10011** [reply]
Sure, I'll look into making one this Weekend!

**Reply by Jojo10011** [reply]
Just found the time to do it! Here you go: https://github.com/mathoudebine/turing-smart-screen-python/discussions/276

**Reply by Freekers** [reply]
Thank you very much, looks awesome!

**Comment by mathoudebine** [comment]
Just in case other people want to use this theme: there is an issue with the polling interval for the GPU data in this theme.
The `GPU` section from the `theme.yaml` file must contain an `INTERVAL` in seconds:
Edit the theme from:
![image](https://github.com/user-attachments/assets/02b1101f-3e24-4c89-9518-7b51828d51e7)
to:
![image](https://github.com/user-attachments/assets/4c0eef49-cc19-4773-9ea0-76880f10a783)
Be careful of the identation!

**Reply by oopiicaa** [reply]
What kind of issue exactly?

**Reply by Felipe-martins-s** [reply]
DUDE! ty so much, I was strugling to find where was the problem that mine wasn't showing the GPU % and Temp, just changed this and worked INSTANTLY even in the preview!

**Comment by eragonhnn** [comment]
I am using Turning Smart Screen 3.5 inch version, how to add this?

**Reply by DJeanS03** [reply]
it achieved?
